examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,524 @@
+{
```

Comment:
<p>acc_fn/acc_val does not instill fear. You might want to cast a more fiendish spell like `accbreitfiskous!" Let them quake with fear!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>